sway-core/src/type_system/ast_elements/type_parameter.rs

```diff
@@ -562,7 +562,8 @@ impl TypeParameter {
                                     trait_types_and_names: concrete_trait_type_ids.iter().map(|t| (engines.help_out(t.0).to_string(), t.1.clone())).collect::<Vec<_>>()
                                 }));
                             }
-                            Ordering::Less => {}
+                            Ordering::Less => {
+                            }
```

Comment:
Are any of the changes in this file the result of the formatter or what? none of it seems necessary or desirable.

Reply:
Yeah. They are comming from `cargo fmt`. Not sure why...

Reply:
Solved.